easybuild/easyconfigs/m/MRCPP/MRCPP-1.3.6-foss-2020a.eb

```
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'MRCPP'
+version = '1.3.6'
+
+homepage = 'mrcpp.readthedocs.io'
```

It is customary for `homepage` to have the protocol explicitly defined. It is not currently checked by our tests but we should change it. 
```suggestion
homepage = 'https://mrcpp.readthedocs.io'
```